app/infrastructure/router.go

```diff
@@ -76,6 +76,7 @@ func Route(connm *sql.DB, connr *redis.Client, l domain.Logger) goblin.Router {
 	r.Methods(http.MethodGet).Use(mw.Auth).Handler(`/private/tags`, tagController.IndexPrivate())
 	r.Methods(http.MethodGet).Use(mw.Auth).Handler(`/private/tags/:id`, tagController.ShowPrivate())
 	r.Methods(http.MethodPost).Use(mw.Auth).Handler(`/private/tags`, tagController.StorePrivate())
+	r.Methods(http.MethodPatch).Use(mw.Auth).Handler(`/private/tags/:id`, tagController.UpdatePrivate())
```

Comment:
OMG
It had been deleted by mistake.
cf. https://github.com/bmf-san/gobel-api/commit/6dc69881e6c0a22c2fc47f6b493bb7886cb8bf5d